src/main/java/net/glowstone/util/OpenCompute.java

```diff
@@ -36,15 +49,18 @@ public static CLProgram getProgram(String name) {
                         programs.put(name, program);
                         return program;
                     } catch (IOException ex) {
-                        GlowServer.logger.log(Level.WARNING, "Could not load custom OpenCL program. Trying builtins.", ex);
+                        GlowServer.logger.log(Level.WARNING,
+                                "Could not load custom OpenCL program. Trying builtins.", ex);
```

Comment:
Could you remove the "Trying builtins" part, as that seems to be false.

Reply:
Done.